contracts/hardhat.config.ts

```diff
@@ -64,7 +64,7 @@ const config: HardhatUserConfig = {
       saveDeployments: true,
       tags: ["staging", "sender", "layer2"],
       companionNetworks: {
-        receiver: "goerli",
+        receiver: "chiado",
```

Comment:
Hmm we'll need to find a better solution, that won't scale. Here the ArbiGoerli->Gerli deployment is certainly affected.

Reply:
Let's look for a better solution later in a separate branch.

Reply:
On it!